bin/cqlsh.py

```diff
@@ -96,7 +96,7 @@
 # >>> webbrowser._tryorder
 # >>> webbrowser._browser
 #
-if len(webbrowser._tryorder) == 0:
+if webbrowser._tryorder is None or len(webbrowser._tryorder) == 0:
```

Comment:
Can you just call `webbrowser.get()` and leave the rest of the code as-is? This will initialize the `_tryorder` variable.

Reply:
in a Docker testing environment it's likely that no default browser is installed and so you could end up in https://github.com/python/cpython/blob/3.7/Lib/webbrowser.py#L65, so I'm +1 with the fix here

Reply:
I tried `webbroser.get()` and ran in docker container, and I hit the error mentioned. So I think it is safer to check if it is `None`.